superset-frontend/src/views/CRUD/data/database/DatabaseModal/DatabaseConnectionForm.tsx

```diff
@@ -44,9 +44,9 @@ export const FormFieldOrder = [
   'username',
   'password',
   'database_name',
+  'encryption',
   'credentials_info',
   'query',
-  'encryption',
```

Comment:
encryption should be below query right?

Reply:
Oops! Fixed in [`this commit`](https://github.com/apache/superset/pull/15234/commits/d9ec9e69a9c6dad602cd52329b22d3e25ac1890e).